device/bgp/bgpexcommunity.yaml

```diff
@@ -0,0 +1,55 @@
+components:
+  schemas:
+    Bgp.ExCommunity:
+      description: >-
+        The Extended Communities Attribute is a transitive optional BGP
+        attribute, with the Type Code 16. Community and Extended Communities 
+        attributes to trigger routing decisions, such as acceptance, rejection, 
+        preference, or redistribution. An extended community is an 8-Bytes value.
+        It is also divided into two main parts. The first 2 Bytes of the community
+        encode a type and sub-type fields and the last 6 Bytes carry a unique set of data in a 
+        format defined by the type and sub-type field. Extended communities provide a larger 
+        range for grouping or categorizing communities.
```

Comment:
The description of extended comm being 8-byte value should be modified. There is no length specified for the ext community. Some ext comm may be 8-byte length.

Reply:
Please ignore above comment . RFC 4360 mentions ext community length to be 8 octets.